apps/angular/projection/src/app/ui/card/card.component.ts

```diff
@@ -40,22 +43,15 @@ import { ListItemComponent } from '../list-item/list-item.component';
   imports: [NgIf, NgFor, ListItemComponent],
 })
 export class CardComponent {
-  @Input() list: any[] | null = null;
+  @Input() list!: any[];
   @Input() type!: CardType;
   @Input() customClass = '';
 
   CardType = CardType;
 
-  constructor(
-    private teacherStore: TeacherStore,
-    private studentStore: StudentStore,
-  ) {}
+  constructor(private cardService: CardService) {}
```

Comment:
nice try, but the goal is to create a dumb component. use `ng-content`, `ng-template-outlet` to project content from the parent

CardType should be deleted at the end